source/configuration_manual/config_file/config_variables.rst

```diff
@@ -134,6 +150,14 @@ Login variables:
 | %k       | ssl_security | TLS session security string. If HAProxy is configured and it    |
 |          |              | terminated the TLS connection, contains "(proxied)".            |
 +----------+--------------+-----------------------------------------------------------------+
+|          | lip          | Deprecated version of %{local_ip}                               |
+|          |              |                                                                 |
+|          |              | .. deprecated:: v2.3.14                                         |
++----------+----------------+---------------------------------------------------------------+
+|          | rip          | Deprecated version of %{remote_ip}                              |
+|          |              |                                                                 |
```

Comment:
Wrong space / - counts in the tables.

Reply:
Fixed. Also notices some other formatting issues. Those should be fixed now as well.